Add note related render use magento bindings (#7969)
magento/devdocs: src/guides/v2.3/ui_comp_guide/concepts/magento-bindings.md

Within HTML templates, Magento gives you the option of using a binding syntax that is simpler and easier to read and write than the standard Knockout binding syntax. The following code snippets help make the comparison.

### Knockout native syntax

```html
<!-- ko if: isVisible -->
    <div class="someClass">
        <!-- ko i18n: 'Some translatable message!' --><!-- /ko -->
        <span data-bind="html: content"></span>

>         <span data-bind="i18n: 'translatable text'"></span>

Reviewer: Seems like this part should be also added in magento version

Author: i have reverted back to origin for maintain backward compatible

If you use the Magento syntax, the Magento wrapper replaces the Magento syntax with the matching Knockout comments or `data-bind` attributes during the loading of the HTML template.

{:.bs-callout-info}

> For better render performance, we recommend using the Knockout native syntax. Magento syntax can render more slowly than the native knockout syntax. When using the Magento syntax, every time the HTML template is loaded for a component, it gets parsed and adjusted accordingly. KnockoutJS knows exactlyt which bindings it should update. Test both ways if performance is critical.

Reviewer: ```suggestion
For better render performance, we recommend using the Knockout native syntax. Magento syntax can render more slowly than the native knockout syntax. When using the Magento syntax, every time the HTML template is loaded for a component, it gets parsed and adjusted accordingly. KnockoutJS knows exactly which bindings it should update. Test both ways if performance is critical.
```

Reviewer: @dobooth could you fix this also?

Reviewer 2: Oops

Reviewer 2: looks like I did already

Author: i have fixed typo

If you use the Magento syntax, the Magento wrapper replaces the Magento syntax with the matching Knockout comments or `data-bind` attributes during the loading of the HTML template.

{:.bs-callout-info}

> For render performance we recommended use knockout native syntax. Magento syntax will render slow than native knockout syntax. When use magento syntax every time HTML template is loaded for a component, it gets parsed and adjusted accordingly so that KnockoutJS actually understands which bindings it should apply. Both approaches use native syntax or magento syntax will work fine but in some case if performance is matter so native syntax is the best option

Reviewer 3: ```suggestion
For render performance, we recommend using knockout native syntax. Magento syntax will render slower than native knockout syntax. When using Magento syntax, every time the HTML template is loaded for a component, it gets parsed and adjusted accordingly so that KnockoutJS actually understands which bindings it should apply. Both approaches using native syntax or Magento syntax will work fine but in some cases if performance will matter so native syntax is the advised option.
```

> For render performance, we recommend using knockout native syntax. Magento syntax will render slower than native knockout syntax. When using Magento syntax, every time the HTML template is loaded for a component, it gets parsed and adjusted accordingly so that KnockoutJS actually understands which bindings it should apply. Both approaches using native syntax or Magento syntax will work fine but in some cases if performance will matter so native syntax is the advised option.

Reviewer 3: ```suggestion
For render performance, we recommend using knockout native syntax. Magento syntax will render slower than native knockout syntax. When using Magento syntax, every time the HTML template is loaded for a component, it gets parsed and adjusted accordingly so that KnockoutJS actually understands which bindings it should apply. Both approaches using native syntax or Magento syntax will work fine, but in some cases performance will matter so native syntax is the advised option.
```
